docs/known_issues.md

```diff
@@ -1,79 +1,104 @@
+
+- **Error**:
+  > TLS handshake timeout
+
+- **Cause**: This is due to the MTU settings of the underlying GRE tunnel used for public internet access.
+
+- **Workaround**: Set MTU of the public interface in bastion to 1450.
+
+  Let's say env3 is the public interface then set MTU to 1450 by running the command: `ifconfig env3 mtu 1450`
+
+## Terraform apply fails with these errors
+
+- **Error**:
+  > module.install.null_resource.install (remote-exec): information.
+module.install.null_resource.install (remote-exec): changed: [192.168.25.12] => {"ansible_facts": {"discovered_interpreter_python": "/usr/libexec/platform-python"}, "changed": true, "cmd": "if lsmod|grep -q 'ibmveth'; then\n  sudo sysctl -w net.ipv4.route.min_pmtu=1450;\n  sudo sysctl -w net.ipv4.ip_no_pmtu_disc=1;\n  echo 'net.ipv4.route.min_pmtu = 1450' | sudo tee --append /etc/sysctl.d/88-sysctl.conf > /dev/null;\n  echo 'net.ipv4.ip_no_pmtu_disc = 1' | sudo tee --append /etc/sysctl.d/88-sysctl.conf > /dev/null;\nfi\n", "delta": "0:00:00.078644", "end": "2020-09-18 16:25:46.414601", "rc": 0, "start": "2020-09-18 16:25:46.335957", "stderr": "", "stderr_lines": [], "stdout": "net.ipv4.route.min_pmtu = 1450\nnet.ipv4.ip_no_pmtu_disc = 1", "stdout_lines": ["net.ipv4.route.min_pmtu = 1450", "net.ipv4.ip_no_pmtu_disc = 1"]}
+  >
+  > module.install.null_resource.install (remote-exec): NO MORE HOSTS LEFT *************************************************************
+  >
+  > module.install.null_resource.install (remote-exec): PLAY RECAP *********************************************************************
+  > module.install.null_resource.install (remote-exec): 192.168.25.105             : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+  > module.install.null_resource.install (remote-exec): 192.168.25.12              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+  > module.install.null_resource.install (remote-exec): 192.168.25.121             : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+  > module.install.null_resource.install (remote-exec): 192.168.25.15              : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+  > module.install.null_resource.install (remote-exec): 192.168.25.235             : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+  > module.install.null_resource.install (remote-exec): 192.168.25.39              : ok=1    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0
+  > module.install.null_resource.install (remote-exec): 192.168.25.5               : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+  > module.install.null_resource.install (remote-exec): 192.168.25.82              : ok=28   changed=22   unreachable=0    failed=0    skipped=11   rescued=0    ignored=0
+
```

Comment:
Can we put some additional empty lines to make the distinction between different types of error?
Currently it looks like the entire message is part of same error..

Reply:
Adding bullet points...